packages/web-console/src/store/Telemetry/epics.ts

```diff
@@ -132,13 +132,43 @@ export const startTelemetry: Epic<StoreAction, TelemetryAction, StoreShape> = (
       if (remoteConfig?.lastUpdated) {
         return from(
           quest.queryRaw(
-            `SELECT cast(created as long), event, origin
-                FROM ${TelemetryTable.MAIN}
-                WHERE created > '${new Date(
-                  remoteConfig.lastUpdated,
-                ).toISOString()}'
-                LIMIT -10000
-            `,
+            `with tel as (
+             SELECT cast(created as long), event, origin
+             FROM ${TelemetryTable.MAIN}
+             WHERE created > '${new Date(
+              remoteConfig.lastUpdated,
+             ).toISOString()}'
```

Comment:
`new Date` part is repeated throghout, would be nice to define it once and reuse

Reply:
Ok, changed